docs/sdk/server-side-sdks/dotnet/dotnet-usage.md

```diff
@@ -107,3 +107,30 @@ To further enable various deployment configurations - we provide a background pr
 or the SDK is not able to make requests to the DevCycle API directly. The installation and setup process for the proxy can be found here: https://github.com/DevCycleHQ/sdk-proxy#readme.
 
 See the [SDK Proxy](../../sdk-proxy/index.md) section for more information.
+
+## Enabling Beta Realtime Updates
+
+:::warning
+This feature is in beta, and may not function as expected. Please ensure that you have the latest version of the SDK.
+:::
+
+This functionality will reduce the number of polling requests that are made to the DevCycle Config CDN, and instead will
+use a long-lived HTTP connection (Server Sent Events) to receive updates when there is a new config available.
+This reduces outbound network traffic, as well as optimizes the SDK for efficiency.
+
+To enable Beta Realtime Updates, pass in the `enableBetaRealtimeUpdates` option to the SDK initialization:
+
+```csharp
+    client = new DevCycleLocalClientBuilder()
+        .SetSDKKey("<DEVCYCLE_SERVER_SDK_KEY>")
+        .SetOptions(new DevCycleLocalOptions(enableBetaRealtimeUpdates: true, eventFlushIntervalMs: 60000))
+        .SetInitializedSubscriber((o, e) => {
+            if (e.Success) {
+                ClientInitialized();
+            } else {
+                Console.WriteLine($"Client did not initialize. Errors: {e.Errors}");
+            }
+        })
```

Comment:
I would remove this if we're not fully specifying all the data/functions it calls.